Good-Bids/goodbids
# [#462] Nonprofit Net Payment Terms

# Summary

This PR adds a new field to the Network Admin > Sites > Details page that allows Super Admins to specify Net Payment Terms for that Nonprofit, which will override the default Net Payment Terms coming from the config. By leaving it blank, the value will fall back to the default config (shown in the field placeholder).

## Issues

* #462 
* #622

## Testing Instructions

1. Update the Net Payment Terms for a Nonprofit.
2. Complete an Auction with a value to generate an Invoice
3. The invoice should be generated with the payment terms set for that specific Nonprofit

## Screenshots

<img width="487" alt="Screenshot 2024-03-08 at 11 02 32 AM" src="https://github.com/Good-Bids/goodbids/assets/122309362/7470ef33-d561-4064-ac03-7d55feb8bbd7">
<img width="547" alt="Screenshot 2024-03-08 at 11 02 16 AM" src="https://github.com/Good-Bids/goodbids/assets/122309362/087244ee-54aa-4e16-9595-78ee8883fb07">
